content/en/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md

```diff
@@ -72,7 +72,7 @@ policies using an example application.
 ## Deploying Cilium for Production Use
 
 For detailed instructions around deploying Cilium for production, see:
-[Cilium Kubernetes Installation Guide](https://docs.cilium.io/en/stable/kubernetes/intro/)
+[Cilium Kubernetes Installation Guide](https://docs.cilium.io/en/stable/concepts/kubernetes/intro/)
```

Comment:
@elvinjgalarza thanks for fixing this link and welcome to the community! 
/lgtm

Reply:
of course! thank you for the opportunity :)